### ISSUE TITLE
npm version is not 1.1.4

### ISSUE THREAD

**Issue by tbslchnr** [LeaVerou/awesomplete#17192]
Hi there, 
the version 1.1.4 provided by npm is not equal to 1.1.4 of GitHub.
(e.g. aira-expanded functionality is missing)
Any solutions ?

**Comment by KittyGiraudel** [comment]
I can confirm as well. Despite having 1.1.4 installed from npm, some accessibility fixes are not present in the code that is shipped. See below a screenshot of the `_.ITEM` function which is missing `role="option"` for instance.

<img width="1440" alt="Screen Shot 2019-08-16 at 4 36 08 PM" src="https://user-images.githubusercontent.com/1889710/63175241-e80cb700-c043-11e9-87d1-2b41892ceccd.png">

Is there anything we can do to help publish the lastest version to npm please? :)

**Comment by LeaVerou** [comment]
@HugoGiraudel This is really weird. If you want to help, you could investigate what's the latest commit in the npm version, that might give us a hint as to what's going on.

[I do see there are 4 commits since 1.1.4](https://github.com/LeaVerou/awesomplete/compare/v1.1.4...gh-pages), perhaps it's these that are not included? I can make a new release if so.

**Comment by dge808** [comment]
@LeaVerou hello, thank you for your answer ! That would be great if you could do a new release. 
It seems that this issue is about these four commits you mentioned. I think we've been a little confused because the version used in demo and code available in the repo isn't the same that is available on npm.
Both `role="option"` and `aira-expanded` were added at the last commit `accessibility fixes`.

**Comment by LeaVerou** [comment]
Done! Please lmk if issue is not fixed.
